pom.xml

```diff
@@ -1201,4 +1201,34 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>tests-with-dependencies</id>
```

Comment:
Let's add an explanation here of how to use this (basically copy/paste from the description).